app/task_routes.py

```diff
@@ -0,0 +1,110 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from datetime import datetime
+from app import db
+import os
+from slack import WebClient
+
+task_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+def validate_task(task_id):
+    try:
+        task_id = int(task_id)
+    except:
+        abort(make_response({"message":f"Task {task_id} invalid"}, 400))
+    
+    task = Task.query.get(task_id)
+    
+    if not task:
+        abort(make_response({"message":f"Task {task_id} not found"}, 404))
+    return task
+
+#CREATE TASK
+
+@task_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+
+    try:
+        new_task = Task.from_dict(request_body)
+        new_task.description == "Test Description"
+        new_task.completed_at == None
+    except:    
+        abort(make_response({"details": "Invalid data"}, 400))
+
+    db.session.add(new_task)
+    db.session.commit()
+    
+    return {"task":new_task.to_dict()}, 201
+
+@task_bp.route("", methods=["GET"])
+def get_all_tasks():
+    title_query = request.args.get("sort")
+    if title_query:
+        if title_query == "asc":
+            tasks = Task.query.order_by(Task.title.asc())
+        elif title_query == "desc":
+            tasks = Task.query.order_by(Task.title.desc())
+    else:
+        tasks = Task.query.all()
+
+    task_response = [Task.to_dict(task) for task in tasks]
```

Comment:
Great work implementing sorting with query params and using a list comprehension here!